package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "nebenan-styleguide",
+  "version": "0.0.0",
+  "license": "MIT",
+  "readmeFilename": "README.md",
+  "repository": "goodhood-eu/styleguide",
+  "bugs": "https://github.com/goodhood-eu/styleguide/issues",
+  "scripts": {
+    "preinstall": "scripts/for-each-package npm install",
```

Comment:
lerna?

Reply:
Moved to favro ticket: https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-31502